superset-frontend/spec/javascripts/dashboard/components/DashboardBuilder_spec.jsx

```diff
@@ -128,10 +128,11 @@ describe('DashboardBuilder', () => {
     expect(parentSize.find(Tabs.TabPane)).toHaveLength(2);
   });
 
-  it('should set animated=true on Tabs for perf', () => {
+  it('should have default animated=true on Tabs for perf', () => {
     const wrapper = setup({ dashboardLayout: undoableDashboardLayoutWithTabs });
     const tabProps = wrapper.find(ParentSize).find(Tabs).props();
-    expect(tabProps.animated).toEqual({ inkBar: true, tabPane: false });
+    console.log('tabProps', tabProps)
```

Comment:
Probably want to remove this console log. There CI linter is failing on the PR - this is likely one of the issues that needs addressing.